packer-task/template.json.pkr.hcl

```diff
@@ -0,0 +1,49 @@
+packer {
+  required_plugins {
+    googlecompute = {
+      source  = "github.com/hashicorp/googlecompute"
+      version = "~> 1"
```

Comment:
Use more restricted version of providers, it's best practices. Your code can not be executed in the future if you use not restricted versions.
Example:
packer {
  required_plugins {
    googlecompute = {
      source  = "github.com/hashicorp/googlecompute"
      version = "1.1.4"
    }
  }
}